salt/modules/boto_secgroup.py

```diff
@@ -255,8 +273,7 @@ def convert_to_group_ids(groups, vpc_id=None, region=None, key=None, keyid=None,
         else:
             log.debug('calling boto_secgroup.get_group_id for'
                       ' group name {0}'.format(group))
-            group_id = get_group_id(group, vpc_id, region, key, keyid,
-                                    profile, vpc_name)
+            group_id = get_group_id(group, vpc_id, vpc_name, region, key, keyid, profile)
```

Comment:
I had actually added the args in a branch WITHOUT them (e.g. no vpc_name)
but by the time I went to merge they'd been added independently in
different locations...  I ended up having to move stuff around to merge and
likely missed some cases.

Thanks again


On Tue, Oct 6, 2015 at 6:36 PM, Thomas Williams <bif.bam@gmail.com> wrote:

> Oook, I thought I had done all of those - thanks for catching :)
>
> t.
>
> On Tue, Oct 6, 2015 at 6:35 PM, Ryan Lane <notifications@github.com>
> wrote:
>
>> In salt/modules/boto_secgroup.py
>> <https://github.com/saltstack/salt/pull/27728#discussion_r41331623>:
>>
>> > @@ -255,8 +273,7 @@ def convert_to_group_ids(groups, vpc_id=None, region=None, key=None, keyid=None,
>> >          else:
>> >              log.debug('calling boto_secgroup.get_group_id for'
>> >                        ' group name {0}'.format(group))
>> > -            group_id = get_group_id(group, vpc_id, region, key, keyid,
>> > -                                    profile, vpc_name)
>> > +            group_id = get_group_id(group, vpc_id, vpc_name, region, key, keyid, profile)
>>
>> Same note for all other locations where you're calling a function where
>> you've moved around the args.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/saltstack/salt/pull/27728/files#r41331623>.
>>
>
>